### DISCUSSION TITLE
setScaleMode

### DISCUSSION THREAD

**Discussion by embraceyouting** [Tencent/libpag#2376] [General] [answered]
官网文档写得一点也不详细，很多地方没有写具体怎么用，还得自己找
npm i libpag 之后需要到node_moudles/libpag/lib将libpag.wasm文件复制到public文件夹下
`import * as PAG from "libpag"`
`pagView.setScaleMode(PAG.types.PAGScaleMode.Stretch);`

**Comment by kevingpqi123** [comment, marked as answer]
https://pag.art/docs/pag-fillmode.html
关于 ScaleMode 有哪里不清晰的，就是一个接口调用

**Reply by Mr-xue** [reply]
人家问的web端的，你文档贴的都是原生端的。 web端的文档还有错的地方，有的api按照文档调不通，还得看源码才知道怎么用。服了
